docs/source/integrations/plugins-event-reference.mdx

```diff
@@ -487,6 +487,8 @@ if the GraphQL operation encounters one or more errors.
 
 (If the operation uses [incremental delivery](../workflow/requests#incremental-delivery-experimental) directives such as `@defer`, `willSendResponse` is called before the *initial* payload is sent; you can use `willSendSubsequentPayload` to find out when more payloads will be sent.)
 
+This is also the place you can mutate the GraphQL response in-place, like adding values to the `extensions` or merging responses from subgraphs with conditional logic.
```

Comment:
I like having this addition in general. I don't know if talking about subgraphs in this Apollo Server context makes sense though (I'm not sure what the use case you're describing is).

Reply:
I suppose I can remove the subgraph reference here. That was more a callout to Gateway